docs/src/validator/best-practices/monitoring.md

```diff
@@ -0,0 +1,83 @@
+
+#### Send a Message to The Bot
+
+Find the bot in Telegram and send it the following message: `/start`. Messaging the bot will help you later when looking for the bot chatroom id.
+
+#### Create Telegram Group
+
+In Telegram, click on the new message icon and then select new group. Find your newly created bot and add the bot to the group. Next, name the group whatever you'd like.
+
+#### Set Environment Variables For Watchtower
+
+Now that you have a bot setup, you will need to set the environment variables for the bot so that watchtower can send notifications.
+
+First, recall the chat message that you got from _@BotFather_. In the message, there was an HTTP API token for your bot. The token will have this format: `389178471:MMTKMrnZB4ErUzJmuFIXTKE6DupLSgoa7h4o`. You will use that token to set the `TELEGRAM_BOT_TOKEN` environment variable. In the terminal where you plan to run `solana-watchtower`, run the following:
+
+```
+export TELEGRAM_BOT_TOKEN=<HTTP API Token>
+```
+
+Next, you need the chat id for your group so that `solana-watcher` knows where to send the message. First, send a message to your bot in the chat group that you created. Something like `@newvalidatorbot hello`. Next, in your browser, got to `https://api.telegram.org/bot<HTTP API Token>/getUpdates`. Make sure to replace `<HTTP API TOKEN>` with your API token that you got in the _@BotFather_ message. Also make sure that you include the word `bot` in the URL before the API token. Make the request in the browser.
+
+The response should be in JSON. Search for the string `"chat":` in the JSON. The `id` value of that chat is your `TELEGRAM_CHAT_ID`. It will be a negative number like: `-781559558`. Remember to include the negative sign! If you cannot find `"chat":` in the JSON, then you may have to remove the bot from your chat group and add it again.
+
+With your Telegram chat id in hand, export the environment variable where you plan to run `solana-watchtower`:
+
+```
+export TELEGRAM_CHAT_ID=<negative chat id number>
+```
+
+#### Restart solana-watchtower
+
+Once your environment variables are set, restart `solana-watchtower`. You should see output about your validator.
+
+To test that your Telegram configuration is working properly, you could stop your validator briefly until it is labeled as delinquent. Up to a minute after the validator is delinquent, you should receive a message in the Telegram group from your bot. Start the validator again and verify that you get another message in your Telegram group from the bot. The message should say `all clear`.
```

Comment:
Love the monitoring addition. I wonder if we should include some third party monitoring such as stakewiz alerts, etc

Reply:
Always good to have multiple monitoring solutions especially in the case of someone running watchtower on their validator hardware and the whole machine goes offline.